docs/tutorials/standalone/Soil/freezing_front.jl

```diff
@@ -106,7 +106,7 @@ hydrology_cm = vanGenuchten{FT}(; α = vg_α, n = vg_n);
 #hcm = BrooksCorey(;ψb = ψb, c = c);
 θ_r = FT(0.05)
 ν_ss_om = FT(0.3)
-ν_ss_quartz = FT(0.7)
+ν_ss_quartz = FT(0.3)
```

Comment:
Why is this changed? I also noticed it has value FT(1) in evaporation.jl. Should it be unified across files?

Reply:
I changed it back - I dont think we have measurements for it, so we dont really know what it will be, but I also dont think it makes a big difference. so might as well keep it the same.

In general this parameter is soil specific (clay vs sand, etc) , so it is OK for it to change across files if those files represent different experiments.